UI/ChatGPT/ChatGPT/ChatGPT/Services/ChatService.cs

```diff
@@ -0,0 +1,93 @@
+using ChatGPT.Business;
+using OpenAI.Managers;
+using OpenAI;
+using System.Runtime.CompilerServices;
+using OpenAI.ObjectModels.RequestModels;
+using OpenAI.ObjectModels;
+using System.Text;
+using OpenAI.ObjectModels.ResponseModels;
+
+namespace ChatGPT.Services;
+public class ChatService : IChatService
+{
+	private OpenAIService _client;
+
+	public ChatService(IOptions<AppConfig> appConfig)
+	{
+		var apiKey = appConfig.Value.ApiKey;
+
+		_client = new OpenAIService(
+			new OpenAiOptions()
+			{
+				ApiKey = apiKey
+			});
+	}
+
+	public async ValueTask<ChatResponse> AskAsync(string prompt)
+	{
+
+		var result = await _client.ChatCompletion.CreateCompletion(new ChatCompletionCreateRequest()
+		{
+			Messages = new List<ChatMessage>
+			{
+				ChatMessage.FromUser(prompt)
+			},
+			Model = Models.Gpt_3_5_Turbo
+		});
+
+		if (result.Successful)
+		{
+			var _result = result.Choices.Select(choice => choice.Message.Content);
+			return new ChatResponse(Status.Value, string.Join("", _result));
```

Comment:
```suggestion
			var result = result.Choices.Select(choice => choice.Message.Content);
			return new ChatResponse(Status.Value, string.Join("", result));
```

(Will conflict with the `var result = await _client.ChatCompletion.CreateCompletion` ... we need to change one name, but not prefix it `_`)

Reply:
Done in next commits.